Pod/Classes/OAStackView.h

```diff
@@ -84,7 +84,7 @@ typedef NS_ENUM(NSInteger, OAStackViewAlignment) {
 NS_ASSUME_NONNULL_BEGIN
 @interface OAStackView : UIView
 
-@property(nonatomic,readonly,copy) NSArray *arrangedSubviews;
+@property(nonatomic,readonly,copy) NSMutableArray *arrangedSubviews;
```

Comment:
I think the idea was to replicate UIStackView API, and in Apple docs `arrangedSubviews` is `NSArray`.